app-modules/interaction/src/Filament/Concerns/HasManyMorphedInteractionsTrait.php

```diff
@@ -121,10 +133,6 @@ public function table(Table $table): Table
                     ->toggleable(isToggledHiddenByDefault: true),
                 TextColumn::make('relation.name')
```

Comment:
You will also want to check if Product wants Outcome and Relation still. They may not have realized these were here because it's toggled hidden by default.